tests/integration_tests/modules/test_mounts.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+
+USER_DATA = """\
+#cloud-config
+swap:
+  filename: /swap.img
+  size: 10000000
+  maxsize: 10000000
+"""
+
+
+def _setup_fallocate_instance(instance):
+    instance.execute('swapoff -a')
+    instance.execute('rm -r /swap.img')
+    instance.execute("echo 'whoops' > /usr/bin/fallocate")
+
+
+class TestFallocateFallback:
+    @pytest.mark.user_data(USER_DATA)
+    @pytest.mark.instance_setup.with_args(_setup_fallocate_instance)
```

Comment:
Unfortunately you can't use a callable arg for a mark without specifying `with_args`